Files.py

```diff
@@ -0,0 +1,130 @@
+from abc import ABC, abstractmethod
+import Users
+
+
+class File(ABC):
+    """
+    Abstract class for files.
+    name: name of the file.
+    """
+    @abstractmethod
+    def __init__(self, name: str):
+        self.name = name
+
+
+class Directory(File):
+    """
+    Class for directories.
+    name: name of the directory.
+    """
+
+    def __init__(self, file_name: str):
+        super().__init__(file_name)
+        self.files = []
+
+    def add_file(self, file) -> None:
+        """
+        Add a file to the directory.
+        :param file: File to add.
+        :return: None.
+        """
+    
+        if file not in self.files:
+            self.files.append(file)
+        
+    def delete_file(self, file_name: str):
+        """
+        Receives file's name and delete it if it is exist.
+        :param file_name: File's name.
+        :return: If delete file.
+        """
+        for file in self.files:
+            if file.get_name() == file_name:
+                self.files.remove(file)
+                return True
+        return False
+
+    def __str__(self) -> str:
+        """
+        Return a string representation of the directory.
+        :return: String representation of the directory.
+        and the files in it.
+        """
+        ret = f"{self.name}:\n"
+        for file in self.files:
+            ret += file.__str__() + ", "
+
+        return ret
+
+    def get_file(self, file_name: str):
+        """
+        Receives file's name and return it if it is exist.
+        :param file_name: File's name.
+        :return: One of the possible types of file if it is exist. Else, None.
+        """
+        for file in self.files:
```

Comment:
Use filter and next - That way you dont have to return None - if there is no file that matches the condition, an exception is thrown, Or you either can pass a default value to the next function